docs/install.html

```
@@ -0,0 +1,249 @@
+                </div>                <div class="verification">
+                    <h2>✅ Verify Installation</h2>
+                    <p>After installation, verify that httprunner is working correctly:</p>
+                    <div class="code-block">
+                        <code># Check version
+httprunner --version
+
+# Show help
+httprunner --help
+
+# Test with a simple request
+echo "GET https://httpbin.org/get" > test.http
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)